docs/spectator/core/meters/timer.md

```diff
@@ -29,6 +30,127 @@ Long Task Timer:
 
 ![Long Task Timer](../../../images/duration_timer.png)
 
+[Percentile Timer]: percentile-timer.md
+## Querying
+
+!!! Note
+    Timers report summarized statistics about the measurements for a time window
+    including the `totalTime`, `count`, `max` and `totalOfSquares`. If you were to simply query for
+    the name of your timer via
+
+    ```
+    nf.cluster,foo,:eq,name,http.req.latency,:eq,:and
+    ```
+
+    you would get a nonsense value that is the sum of the reported statistics.
+
+When querying the results of a timer, use one of the ASL functions below to generate
+a useful response.
+
+### Average Measurement (:dist-avg)
```

Comment:
I think this goes into too much detail that is just going to make it more confusing for a new user just trying to use a timer. The finer details of how the average is computed should probably be placed in the reference page for the `:dist-avg` operator. For this section it seems like enough to just advise using `:dist-avg` and being clear it is an average per recorded value, e.g. request.

Reply:
I was kinda wondering why all of that detail was here. I'll move them.